src/ashwface.h

```diff
@@ -292,8 +292,11 @@ class FENAW
 #define FCSL( s) s, sizeof( s)
 #define FCSET( d, s) strSpacePad( FCSL( d), s)
 #define FCGET( s) strTrim( NULL, FCSL( s))
+#if defined(SUPPORT_XMODULE)
 #include "xmodule.h"
 class XASHWAT : public XMODULE
+#endif // SUPPORT_XMODULE
+class XASHWAT
```

Comment:
This needs an #else.  Would not compile if SUPPORT_XMODULE is defined.

In fact, did you confirm that the code will build with #define SUPPORT_XMODULE?

Reply:
Thank you for catching that error. I build the code with #define SUPPORT_XMODULE and the only thing that one needs to add is the xmodule.cpp and xmodule.h on the CMakeList. Once that is added, it should run.